tests/gasket_mininet_test_unit.py

```diff
@@ -401,7 +395,9 @@ def start_dhcp_server(self, host, gw, dns):
         print('* Starting DHCP server on', host, 'at', host.IP(), '\n')
         dhcp_config = '/tmp/%s-udhcpd.conf' % host
         self.make_dhcp_config(dhcp_config, host.defaultIntf(), gw, dns)
-        host.cmd('udhcpd -f', dhcp_config,
+        host.cmd('touch %s/dhcp.leases' % self.tmpdir)
```

Comment:
do these file writes need to be done by the host?

Reply:
Suppose that if the host had a private filesystem they might be? (not sure how private filesystem works)